### ISSUE TITLE
Стр. 182

### ISSUE THREAD

**Issue by Cleverscript** [mflenov/phphacker#3]
Глава 3.18
[chapter3/kaptcha/protect.php](https://github.com/mflenov/phphacker/blob/main/chapter3/kaptcha/protect.php)

`imageline($img, 0, rand(0,24), 130, rand(0,40), $DDDDDD);`

$DDDDDD не определена

**Comment by mflenov** [comment]
Да, желательно было явно задать, в данном случае за счёт отсутствия значения будет использоваться чёрный цвет (0)
